Library/Formula/backupninja.rb

```diff
@@ -1,19 +1,18 @@
-require 'formula'
-
 class Backupninja < Formula
   homepage "https://labs.riseup.net/code/projects/backupninja"
-  head "git://labs.riseup.net/backupninja.git"
   url "https://labs.riseup.net/code/attachments/download/275/backupninja-1.0.1.tar.gz"
-  sha1 "c9b2cef3c289b2b71cc8fd33f8e089a70a11affb"
+  sha256 "10fa5dbcd569a082b8164cd30276dd04a238c7190d836bcba006ea3d1235e525"
+
+  head "https://labs.riseup.net/git/backupninja.git"
 
   depends_on "dialog"
   depends_on "gawk"
 
-  skip_clean 'etc/backup.d'
+  skip_clean "etc/backup.d"
 
   def install
     system "./configure", "--prefix=#{prefix}"
     system "make"
-    system "make install"
+    system "make", "install"
   end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!